src/test/java/com/epam/tc/hw3/pageobjects/elements/BaseElements.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.tc.hw3.pageobjects.elements;
+
+import org.openqa.selenium.NoSuchSessionException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public class BaseElements {
+
+    WebDriver webDriver;
+
+    public BaseElements(WebDriver webDriver) {
+        PageFactory.initElements(webDriver, this);
+        this.webDriver = webDriver;
+    }
+
+    public String openURL(String siteURL) {
+        webDriver.navigate().to(siteURL);
+        String actualURL = webDriver.getCurrentUrl();
+        return actualURL;
+    }
+
+    public String getTitle() {
+        String title = webDriver.getTitle();
+        return title;
+    }
+
+    public String getCurrentUrl() {
+        return webDriver.getCurrentUrl();
+    }
```

Comment:
I suppose that these methods should be in `BasePage` class

Reply:
Fixed